netebpfext/net_ebpf_ext.c

```diff
@@ -620,13 +631,16 @@ static NTSTATUS
 _net_ebpf_ext_filter_change_notify(
     FWPS_CALLOUT_NOTIFY_TYPE callout_notification_type, _In_ const GUID* filter_key, _Inout_ const FWPS_FILTER* filter)
 {
+    NET_EBPF_EXT_LOG_ENTRY();
+
     UNREFERENCED_PARAMETER(filter_key);
     if (callout_notification_type == FWPS_CALLOUT_NOTIFY_DELETE_FILTER) {
         net_ebpf_extension_wfp_filter_context_t* filter_context =
             (net_ebpf_extension_wfp_filter_context_t*)(uintptr_t)filter->context;
         DEREFERENCE_FILTER_CONTEXT((filter_context));
     }
 
+    NET_EBPF_EXT_LOG_FUNCTION_SUCCESS();
     return STATUS_SUCCESS;
```

Comment:
can these be compiled into one line?
```suggestion
    return NET_EBPF_EXT_RETURN_NTSTATUS(STATUS_SUCCESS);
```

Reply:
No - does not compile due to the way the macro is defined.

Reply:
What precisely is the problem?  Do we need to fix the macro?